e2e/_suites/metricbeat/features/integrations.feature

```diff
@@ -14,6 +14,44 @@ Examples: Apache
 | apache      | 2.4.12  |
 | apache      | 2.4.20  |
 
+@dropwizard
+Examples: Dropwizard
+| integration | version |
+| dropwizard  | 4.1.2   |
+| dropwizard  | 4.0.0   |
+| dropwizard  | 3.2.6   |
+
+@envoyproxy
+Examples: EnvoyProxy
+| integration | version |
+| envoyproxy  | 1.7.0   |
+| envoyproxy  | 1.12.0  |
+
+@etcd
+Examples: Etcd
+| integration | version |
+| etcd        | 3.2.25  |
+| etcd        | 3.3.10  |
+
+@haproxy
+Examples: HAProxy
+| integration | version |
+| haproxy     | 2.0.9   |
+| haproxy     | 1.9.12  |
+| haproxy     | 1.8.22  |
+| haproxy     | 1.7.12  |
+| haproxy     | 1.6.15  |
+
+@kafka
+@skip
```

Comment:
is this the appropriate way to put in multiple tags?!  Neat, I had thought to put them side by side, I didn't think to stack them.

Also, when putting in a skip a common practice is to put a comment with the relating issue, so it easier to review and un-skip when its fixed (especially if it is forgotten about).

Reply:
I always trust the guy commit causing the line change, in this case the one adding the skip tag